CHANGES.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 1.9.10 (core) / 0.25.10 (libraries)
+
+### New
+
+- Implemented the `AutomationCondition.replace()` function to improve ergonomics of updating sub-conditions.
+- Implemented the `allow()` and `ignore()` methods on boolean Declarative Automation conditions, which will propagate the asset selection down to any contained dep sub-conditions.
```

Comment:
Added new `.allow()` and `.ignore()` methods to the boolean `AutomationCondition` operators, which allow asset selections to be propagated to sub-conditions such as `AutomationCondition.any_deps_match()` and `AutomationCondition.all_deps_match()`.

Reply:
```suggestion
- Added new `.allow()` and `.ignore()` methods to the boolean `AutomationCondition` operators, which allow asset selections to be propagated to sub-conditions such as `AutomationCondition.any_deps_match()` and `AutomationCondition.all_deps_match()`.
```